client/src/styles/EmojiPicker.scss

```diff
@@ -0,0 +1,36 @@
+.emoji-picker {
+  width: 235px;
+  background: #fff;
+  box-sizing: border-box;
+  border-radius: 2px;
+  overflow: auto;
+  z-index: 2;
+  position: absolute;
+  bottom: 44px;
+  background-color: rgb(64, 64, 64);
+  box-shadow: 1px 2px 2px 1px rgba(0,0,0,0.3);
```

Comment:
There are currently no shadows in the channel and I'd like to keep it that way for now. What you could try is a border that's a just tiny bit lighter/darker than the background color (see what fits). If border doesn't look good, go without border and without shadow.